jupyterhub_traefik_proxy/etcd.py

```diff
@@ -58,8 +58,9 @@ def _default_client(self):
             endpoint=str(etcd_service.hostname) + ":" + str(etcd_service.port)
         )
 
-    command = "traefik"
-    traefik_port = traefik_utils.get_port(command)
+    traefik_url = Unicode(
+        "http://127.0.0.1:8000", config=True, help="""The URL of the etcd server"""
```

Comment:
The Proxy base class already has a `public_url` Unicode trait for this information. Let's use that.

We can set it in the proxy fixture with:

```python
proxy = TraefikEtcdProxy(public_url='http://127.0.0.1:10291')
```

Reply:
Fixed, thanks for the feedback, I've totally missed that.